libvmi/os/windows/peparse.c

```diff
@@ -389,12 +389,12 @@ peparse_get_idd_rva(
 
     if(optional_header_type == NULL) {
 
-        if(oh_pe32 != NULL) {
+        if(oh_pe32 != NULL && oh_pe32->number_of_rva_and_sizes >= entry_id) {
```

Comment:
I had to think about this one longer than if it had been `entry_id < oh_pe32->number_of_rva_and_sizes`, but as long as it's consistent within these functions, it can stay this way.